lib/commands/generate.js

```diff
@@ -18,6 +18,7 @@ module.exports = Command.extend({
 
   availableOptions: [
     { name: 'dry-run',       type: Boolean, default: false, aliases: ['d'] },
+    { name: 'skip-prompts',  type: Boolean, default: false, aliases: ['sp'] },
```

Comment:
@Turbo87 how would the `--yes` apply to multiple yes/no prompts?

@stefanpenner I think it's less descriptive, and you have to set it explicitly, i.e. `--skip-prompts` vs `--interactive=false`.